### PR TITLE
bug: set dark theme color for error panel in session detail modal

### PR DESCRIPTION
**Pull request by ironAiken2**
### This PR Resolves [#2408](https://github.com/lablup/backend.ai-webui/issues/2408)

|light|dark|
|---|---|
|||

**Checklist:** (if applicable)

- [ ] Mention to the original issue
- [ ] Documentation
- [ ] Minium required manager version
- [ ] Specific setting for review (eg., KB link, endpoint or how to setup)
- [ ] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

**Comment by ironAiken2** [comment]
* **#2489** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2489?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ironAiken2 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/components/backend-ai-session-list.ts [hunk 1]
```diff
@@ -531,6 +531,13 @@ export default class BackendAISessionList extends BackendAIPage {
         div.usage-items {
           font-size: 10px;
         }
+
+        div.error-detail-panel {
+          border-radius: 4px;
```
Comment (agatha197) [inline src/components/backend-ai-session-list.ts]:
```suggestion
          border-radius: var(--token-borderRadiusSM, 4px);
```

File: src/components/backend-ai-session-list.ts [hunk 2]
```diff
@@ -531,6 +531,13 @@ export default class BackendAISessionList extends BackendAIPage {
...
+          background-color: var(--token-colorBgContainerDisabled);
+          padding: 10px;
+          margin: 10px;
```
Comment (agatha197) [inline src/components/backend-ai-session-list.ts]:
```suggestion
          padding: var(--token-paddingSM, 10px);
          margin: var(--token-marginSM, 10px);
```

**Review by agatha197 — CHANGES_REQUESTED** [review]

**Review by agatha197 — APPROVED** [review]
LGTM

**Pull request merged**